netkan/netkan/download_counter.py

```diff
@@ -113,7 +113,10 @@ def get_result(self, counts: Optional[Dict[str, int]] = None) -> Dict[str, int]:
             if user_repo in self.cache:
                 count = self.cache[user_repo]
                 logging.info('Count for %s is %s', ident, count)
-                counts[ident] = count
+                if ident in counts:
+                    counts[ident] += count
+                else:
+                    counts[ident] = count
```

Comment:
I always feel a bit funny about these kinds of if/else statements in python (and not using dict methods), this would remove the nested if/else and achieve the same outcome.
```suggestion
                counts.update({ident: counts.get(ident, 0) + count})
```

```python
In [1]: ident = 'AwesomeMod'

In [2]: counts = {}

In [3]: count = 3

In [4]: counts.update({ident: counts.get(ident, 0) + count})

In [5]: counts
Out[5]: {'AwesomeMod': 3}

In [6]: counts.update({ident: counts.get(ident, 0) + count})

In [7]: counts
Out[7]: {'AwesomeMod': 6}
```

Reply:
I don't love the if/else either, but I like it better than having to know what `update` does and creating a throwaway `dict` object.

What if we refactor the if/else into an `AddOrSum` method?

Reply:
Found a compromise that does it in one statement without using `update` and an extra `dict`.